services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Utils.kt

```diff
@@ -20,6 +20,9 @@ fun getCurrentTimeUTCAsLocalTime(): LocalDateTime {
 
 fun currentUserId(): String = SecurityContextHolder.getContext().authentication.name
 
+/** Function to validate a given condition. Argument should throw if condition is not met. */
+fun checkCondition(condition: () -> Unit) = condition()
```

Comment:
How is `checkCondition { condition() }` different from `condition()`?

Reply:
The signature looks like it would do something like turn a boolean into an exception, so this might be distracting.

Reply:
Removed checkCondition.